FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/DataProcessor/ParquetDataProcessor.cs

```diff
@@ -26,7 +24,7 @@ namespace Microsoft.Health.AnalyticsConnector.Core.DataProcessor
 {
     public sealed class ParquetDataProcessor : IColumnDataProcessor
     {
-        private readonly ArrowConfiguration _arrowConfiguration;
+        private const int LargeResourceSize = 100 << 20;
```

Comment:
Can we add some comments for this value?

Reply:
Sure.